test/utility/mockApp.ts

```diff
@@ -2,8 +2,8 @@ import * as models from 'powerbi-models';
 
 export interface IApp {
   // Load
-  load(config: models.ILoadConfiguration): Promise<void>;
-  validateLoad(config: models.ILoadConfiguration): Promise<models.IError[]>;
+  load(config: models.IReportLoadConfiguration): Promise<void>;
+  validateLoad(config: models.IReportLoadConfiguration): Promise<models.IError[]>;
```

Comment:
They perform the same function just in different places, although it would probably be clearer to rename `validate` to `validateLoad` so people know it's generic for validating loading, but less generic that `validateReportLoad` which is only for report loads.